tests/phpunit/elementor/app/onboarding/test-features-usage.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Elementor\Tests\Phpunit\Elementor\App\Onboarding;
+
+use Elementor\App\Modules\Onboarding\Features_Usage;
+use ElementorEditorTesting\Elementor_Test_Base;
+
+class Test_Features_Usage extends Elementor_Test_Base {
+
+	public function test_save_onboarding_features() {
+		// Arrange
+		$features_usage = new Features_Usage();
+		$features = [ 'feature_1', 'feature_2' ];
+		$data = json_encode( [
+			'features' => $features,
+		] );
+
+		// Act
+		$result = $features_usage->save_onboarding_features( $data );
+		$option_value = get_option( Features_Usage::ONBOARDING_FEATURES_OPTION );
+
+		// Assert
+		$this->assertEquals( [
+			'status' => 'success',
+			'payload' => [],
+		], $result );
+		$this->assertEquals( $features, $option_value );
+	}
+
+	public function test_register() {
+		// Arrange
+		remove_all_filters( 'elementor/tracker/send_tracking_data_params' );
```

Comment:
Why so you need `remove_all_filters`?

Reply:
Just to have the expected result in assertions